www/amber/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        amberframework amber 0.8.0 v
+
+categories          www devel
+platforms           darwin
+license             MIT
+maintainers         {@conradwt gmail.com:conradwt} openmaintainer
+
+description         Crystal web application framework
+
+long_description    Amber is web application framework implemented in the Crystal \
+                    programming language that makes building applications fast, \
+                    simple, and enjoyable. Get started with quick prototyping, \
+                    less bugs, and blazing fast performance. 
```

Comment:
Not something I'm going to block this for, but the grammar geek in me winces here. "fewer bugs" is correct, because bugs are discrete rather than continuous things.